scripts/tests/test_import_opentextbooks.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+from ..import_open_textbook_library import map_data
+
+
+@pytest.mark.parametrize(
+    "input_data, expected_output",
+    [
+        (
+            # Test case 1: Basic case with all fields present
+            {
```

Comment:
Can you use actual examples from the dataset here? That'll be more robust and catch more errors.